modules/cluster-logging-must-gather-collecting.adoc

```diff
@@ -4,17 +4,17 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="cluster-logging-must-gather-collecting_{context}"]
-= Collecting OpenShift Logging data
+= Collecting {logging} data
 
-You can use the `oc adm must-gather` CLI command to collect information about your {logging}.
+You can use the `oc adm must-gather` CLI command to collect information about {logging}.
 
 .Procedure
 
 To collect {logging} information with `must-gather`:
 
 . Navigate to the directory where you want to store the `must-gather` information.
 
-. Run the `oc adm must-gather` command against the OpenShift Logging image:
+. Run the `oc adm must-gather` command against the {logging} image:
```

Comment:
```suggestion
. Run the `oc adm must-gather` command against the {logging} image:
```

Reply:
{logging} is not defined as anything more than the word "logging". If that's OK by you, it's OK by me. :-D